spec/api/v1/restrooms_spec.rb

```diff
@@ -55,8 +54,8 @@
 
     let(:json) { JSON.parse(response.body) }
 
-    context 'a list of unisex restrooms' do
-      before :each do
+    context 'when requesting a list of unisex restrooms' do
```

Comment:
I changed the wording because of the [`RSpec/ContextWording`](https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspeccontextwording) rule (this rule is also why I changed some `context` blocks to `describe` since it seemed more semantic).

If I'm very off base with the wording I can just disable this rule.

Reply:
I appreciate the more semantic wording! I like it when the error messages read like natural language and make sense as sentences.

And so, I was actually meaning to go back and review these and just make sure the new wordings still captured the intent or explain what the test is for in an accurate way. Wanted to help ensure that could happen. I'll see how involved this all gets... Could be some work, but it seems worth it to get it right now for benefit of use later on down the line.